packages/millicast-viewer-demo/src/viewer.js

```diff
@@ -81,9 +81,6 @@ const newViewer = () => {
         millicastView.configureDRM(drmOptions);
       }
     }
-    let layers = event.data["layers"] !== null ? event.data["layers"] : {};
-    if (event.name === "layers" && Object.keys(layers).length <= 0) {
-    }
```

Comment:
Was this for testing purposes?

Reply:
It's useless now, so remove it from demo